examples/employee-app/src/main/java/com/example/helidon/employee/DBConnection.java

```diff
@@ -0,0 +1,66 @@
+/* Copyright 2019 Oracle and/or its affiliates. All rights reserved. */
+package com.example.helidon.employee;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.Properties;
+
+import oracle.jdbc.pool.OracleDataSource;
+
+public class DBConnection {
+
+    public static final String DB_USERNAME;
+    public static final String DB_USER_PASSWORD;
+    public static final String DB_HOST_URL;
+
+    //Initialize DB Credentials
+    static {

```

Comment:
No need for a static constructor since this class is a Singleton, you should move these in the constructor.

Reply:
This should be fixed now.